packages/react-scripts/utils/createJestConfig.js

```diff
@@ -38,6 +38,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
     transformIgnorePatterns: [
       '[/\\\\]node_modules[/\\\\].+\\.(js|jsx)$'
     ],
+    moduleDirectories: ['node_modules', process.env.NODE_PATH].filter(dir => dir),
```

Comment:
Yes. See https://github.com/facebookincubator/create-react-app/pull/1194 for context.